addon/utils/is-in-viewport.js

```diff
@@ -11,7 +11,7 @@ const defaultTolerance = {
 
 export default function isInViewport(boundingClientRect = {}, height = 0, width = 0, tolerance = defaultTolerance) {
   const { top, left, bottom, right } = boundingClientRect;
-  const tolerances = assign(defaultTolerance, tolerance);
+  const tolerances = assign({}, defaultTolerance, tolerance);
```

Comment:
@miguelcobain this will fail if `assign` falls back to Ember.merge 😢.

You need to do the following:

```js
assign(assign({}, defaultTolerance), tolerance))
```